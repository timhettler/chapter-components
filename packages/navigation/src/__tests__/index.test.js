import React from 'react';
import Component from '../';
import renderer from 'react-test-renderer';

it(`${Component.name} renders correctly`, () => {
  const tree = renderer
    .create(
      <Component
        chapters={[]}
        accessibility={{}}
        label={'All chapters'}
        currentChapter={'04'}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
