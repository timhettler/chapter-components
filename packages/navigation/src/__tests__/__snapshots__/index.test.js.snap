// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders correctly 1`] = `
<nav
  className="navigation"
>
  <div
    className="navigation__bgLayer"
  />
  <button
    aria-expanded={false}
    className="menuButton menuButton--is-light"
    onClick={[Function]}
  >
    <span
      className="menuButton__icon menuButton__icon--primary"
    >
      <svg
        className="icon_style"
        height="100%"
        viewBox="0 0 50 50"
        width="100%"
      >
        <path
          d="M11 1v6H1v42h38v-6h10V8.586L41.414 1H11zm31 3.414L45.586 8H42V4.414zM37 47H3V9h27v7h7v31zm-5-36.586L35.586 14H32v-3.586zM39 41V14.586L31.414 7H13V3h27v7h7v31h-8z"
        />
      </svg>
    </span>
    <span
      className="menuButton__icon menuButton__icon--secondary"
    >
      <svg
        className="icon_style"
        fill="#fff"
        height="100%"
        viewBox="0 0 64 64"
        width="100%"
      >
        <g
          id="Icon-Close"
          transform="translate(381 231)"
        >
          <path
            d="M-370.7-174.7l-2.3-2.3 46-46 2.3 2.3-46 46"
          />
          <path
            d="M-327-174.7l-46-46 2.3-2.3 46 46-2.3 2.3"
          />
        </g>
      </svg>
    </span>
    <div
      className="menuButton__copy"
    >
      <span
        className="menuButton__text menuButton__text--primary"
      >
        All chapters
      </span>
      <span
        className="menuButton__text menuButton__text--secondary"
      >
        All chapters
      </span>
    </div>
  </button>
  <ol
    className="menuList"
  />
</nav>
`;
